sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -256,6 +259,33 @@ private boolean canLogAtLevel(LogLevel logLevel, LogLevel environmentLoggingLeve
         }
     }
 
+    /**
+     * Checking the system log with the preference order of slf4j and environment variable.
+     *
+     * @return The log level.
+     */
+    private void loadSystemLogLevel() {
+        if (rootLogLevel != null) {
+            return;
+        }
+        final StaticLoggerBinder binder = StaticLoggerBinder.getSingleton();
+        if (binder.getLoggerFactory() != null) {
+            if (logger.isDebugEnabled()) {
+                rootLogLevel = LogLevel.VERBOSE;
+            }
+            if (logger.isInfoEnabled()) {
+                rootLogLevel = LogLevel.INFORMATIONAL;
+            }
+            if (logger.isWarnEnabled()) {
+                rootLogLevel = LogLevel.WARNING;
+            }
+            if (logger.isErrorEnabled()) {
+                rootLogLevel = LogLevel.ERROR;
+            }
```

Comment:
Should this be an `if/else if` or `switch` block? Right now this will continue updating the `rootLogLevel` for each conditional that passes and will result with whichever passes last as the `rootLogLevel`.